speech/snippets/multichannel.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Package snippets contains speech examples.
+package snippets
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	// [START imports]
+	"context"
+
+	speech "cloud.google.com/go/speech/apiv1p1beta1"
+	speechpb "google.golang.org/genproto/googleapis/cloud/speech/v1p1beta1"
+	// [END imports]
+)
+
+// TranscribeMultichannel generates a transcript of a multichannel speech file and tags the speech from each channel.
```

Comment:
The comment should be on the line directly above the function declaration and have the same case.

```
// transcribeMultichannel ...
func transcribeMultichannel(...)...
```

Reply:
fixed